core/server/worker/src/main/java/alluxio/worker/grpc/BlockWorkerClientServiceHandler.java

```diff
@@ -147,12 +147,11 @@ public BlockWorkerClientServiceHandler(WorkerProcess workerProcess, FileSystemCo
   }
 
   @Override
-  public void asyncCache(AsyncCacheRequest request,
```

Comment:
keep `asyncCache`, convert `AsyncCacheResponse` to the corresponding `CacheResponse` and passing it to `mBlockWorker.cache()`

Reply:
a little confused, you mean cacheRequest here?